Dockerfile

```diff
@@ -6,8 +6,8 @@ ENV BUILD_PKGS go git
 RUN eve-alpine-deploy.sh
 
 # FIXME: integrate go 1.16 into eve-alpine
-RUN wget https://dl-cdn.alpinelinux.org/alpine/edge/community/$(uname -m)/go-1.16.5-r0.apk -O go.apk &&\
-    apk add ./go.apk
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.14/community' >> /etc/apk/repositories &&\
+  apk --no-cache add -U --upgrade go && go version
```

Comment:
I like moving this from `edge` to `3.14`, and from too locked down to version out to generic. Is this still because [lfedge/eve-alpine is on 3.13](https://github.com/lf-edge/eve/blob/master/pkg/alpine/Dockerfile#L4)?

It still is a bit risky that we are including 3.14 in general. Could we just use it directly? Would the following work?

```sh
apk --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.14/community  add -U --upgrade go
```

Reply:
Comment here if it works, and then we can merge in

Reply:
Good point, thank you for the better variant!
Yes, we are still on 3.13

Reply:
> Comment here if it works, and then we can merge in

It works and builds successfully.